source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -164,9 +274,83 @@ provision the following networking API entities, represented in yaml:
       - subnet_id: "{{ ipv6_subnet.id }}"
 ~~~~~
 
+### Configuring stateless DHCPv6 on the subnet
+
+To configure stateless Dhcpv6 on the subnet, the user should provision
+the following yaml based ansible playbook snippet:
+
+~~~~~
+
+- name: create a network
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a logical switch port
+  os_port:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: lport1
+    network: "{{ ipv6_network.id }}"
+    fixed_ips:
+      - subnet_id: "{{ dhcpv6_subnet.id }}"
+
+- name: create a router
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+
+~~~~~
+
+## UI considerations
+
+Having dynamic address assignment in EL8 requires stateless dhcpv6 subnet
+configuration, which in turn requires an OVN logical router, in order for
+the RA generation logical flows to be installed.
+
+It is outside the scope of this feature to implement the UI changes that allow
+logical routers to be provisioned, and the REST API - or its ansible wrapper
+should be used instead. An ansible example is available
+[here](#configuring-stateless-dhcpv6-on-the-subnet).
+
 ## Testing
 
-Two scenarios are recommended:
-  - single IPv6 subnet
-  - multiple IPv6 subnets connected by router
+Two scenarios are recommended, each with two different configurations, leading
+to four different tests:
+  - single IPv6 subnet using dhcpv6-stateful configuration
+  - single IPv6 subnet using dhcpv6-stateless configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateful configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateless configuration
+
+### Supported options on each ipv6_address_mode
+
+Some of the DHCP options defined in multiple RFCs are supported by OVN,
+and can be leveraged accordingly to enable multiple use cases.
+
+The following table exposes those, and indicates which ipv6_address_mode it
+requires to be implemented.
+
+|        Option Name        | Supported ipv6_address_mode |
```

Comment:
If it would be written like this, it would be transparent to the reader what is expected to be happen, and what will be availible if the guest does not use dhcp:

```
|        Option Name        | dhcpv6-stateful | dhcpv6-stateless |
|:-------------------------:|:----------------------------------:|
| DNS recursive name server |        DHCP     |      DHCP        |  
|     Domain Search List    |        DHCP     |      DHCP        |
|   Network Prefix          |        DHCP     |      RA          |  
|   Network Prefix Length   |        RA?      |      RA          |
|            MTU            |        RA?      |      RA          |
|     Default gateway       |        RA?      |      RA          |
```

Reply:
+1

Reply:
I don't get what 'RA?' means. There's no way to get those through dhcpv6 that I'm aware of.

If that's what you want to express, a '-', or None, or 'No way to get this' is a lot better imho.

Reply:
`RA` means router advertisements.
I don't think that `dhcpv6-stateful` means no router advertisements, I think it just means that the address is managed by a stateful dhcp server, while the `dhcpv6-stateless` just multicasts the network prefix via RA. Additional to the address there are stateless options which might be managed by dhcp or RA in both cases.

Reply:
I didn't understand what you meant with 'RA?' - with question mark.

Reply:
Went with your table.

Done